Jenkinsfile

```diff
@@ -310,6 +312,58 @@ def doCheckInDocker(Map options = [:]) {
     }
 }
 
+def doCheckAlpine(Map options = [:]) {
```

Comment:
Looks like it took CI 18 minutes to compile with musl c. It's not the bottleneck on CI so doesn't matter too much, but  that's around double the time for our other linux builders 😄